exec/java-exec/src/test/java/org/apache/drill/test/PrintingUtils.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.test;
+
+import ch.qos.logback.classic.Level;
+import org.apache.drill.exec.client.LoggingResultsListener;
+import org.apache.drill.exec.util.VectorUtil;
+
+import java.util.function.Supplier;
+
+/**
+ * <p>
+ *   This class contains utility methods to run lambda functions with the necessary {@link org.apache.drill.test.LogFixture}
+ *   boilerplate to print results to stdout for debugging purposes.
+ * </p>
+ *
+ * <p>
+ *   If you need to enable printing for more classes, simply add them to the {@link org.apache.drill.test.LogFixture}
+ *   constructed in {@link #printAndThrow(CheckedSupplier)}.
+ * </p>
+ */
+public final class PrintingUtils {
+  /**
+   * The java standard library does not provide a lambda function interface for funtions that take no arguments,
+   * but that throw an exception. So, we have to define our own here for use in {@link #printAndThrow(CheckedSupplier)}.
+   * @param <T> The return type of the lambda function.
+   * @param <E> The type of exception thrown by the lambda function.
+   */
+  @FunctionalInterface
```

Comment:
Maybe move it to `org.apache.drill.exec.util` class? It maybe used in other classes in future.

Reply:
done